### ISSUE TITLE
Picking an image from storage.

### ISSUE THREAD

**Issue by ibrahim-cse** [omarhurani/flutter_painter#18]
Instead of showing a network image in the initial image view, I want to pick an image from the local storage directory and edit it. Is it possible?

**Comment by omarhurani** [comment]
Yes. As stated in the [background](https://github.com/omarhurani/flutter_painter/blob/master/README.md#background) section of the [README](https://github.com/omarhurani/flutter_painter/blob/master/README.md) file, you can use any [`ImageProvider`](https://api.flutter.dev/flutter/painting/ImageProvider-class.html) with the `ImageProvider.image` extension getter, such as [`FileImage`](https://api.flutter.dev/flutter/painting/FileImage-class.html), [`MemoryImage`](https://api.flutter.dev/flutter/painting/MemoryImage-class.html) or [`NetworkImage`](https://api.flutter.dev/flutter/painting/NetworkImage-class.html).

**Comment by ibrahim-cse** [comment]
Thanks for the quick response. It worked. ✌🏼

**Comment by zolzaya** [comment]
Hey how can I load from `assets` folder. I tried but I can't make `image` to `backgroundDrawable`. Please provide us an example?
